docs/pages/_app.tsx

```diff
@@ -95,7 +95,20 @@ export default function MyApp({ Component, pageProps }: AppProps) {
             inset: 0;
             top: var(--top-nav-height);
             background: var(--primary-bg-color);
-            display: ${navOpen ? "block" : "none"};
+            z-index: 100;
+
+            transition: opacity 0.2s, transform 0.2s;
+            ${navOpen
```

Comment:
I don't think we can do `style jsx` like this (dynamically have certain properties in the string. I think this works since it's equivalent of doing:

```
opacity: ${bool ? 1 : 0};
...
```

But I think this puts in a place where it's easier to run into issues since if we dynamically remove a CSS attribute, styled jsx will get confused. Let's use one of their recommended ways of doing dynamic attributes: https://github.com/vercel/styled-jsx#dynamic-styles